src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -1390,10 +1390,9 @@ public override void WriteProgress(Int64 sourceId, ProgressRecord record)
                 }
                 else
                 {
-                    lock (_instanceLock)
-                    {
-                        HandleIncomingProgressRecord(sourceId, record);
-                    }
+                    // lock (_instanceLock) is moved into HandleIncomingProgressRecord
```

Comment:
This comment should be removed.
It's good for the code-review purposes (i.e. in the commit message) but not in the actual code.